### PR TITLE
doc: add maxuploadtarget to bitcoinf.conf example

### PR DESCRIPTION
**Pull request by apitko**
I reduced mine to 10000 since I was seeing about 40GB+ a day being uploaded and need to utilize my bandwidth for other projects. Running bitcoin-qt on a Raspberry Pi 4 with a gig fiber connection. Would be interested if maxconnections is a better use to throttle than upload limits or if this suggested parameter can cause issues. So far throttling looks much better for me. Thank you for the conf file!

### CODE REVIEW

**Comment by fanquake** [comment]
Note the CI is failing due to:
> ⚠️ User is too new to use Community clusters! Please check with support!

@fkorotkov what is the threshold for new GitHub accounts before the CI will run?

**Comment by prayank23** [comment]
Is this related to reddit post: https://www.reddit.com/r/Bitcoin/comments/m88pad/full_nodes_and_isp_data_caps/ ?

> Running bitcoin-qt on a Raspberry Pi 4 with a gig fiber connection.

Umbrel?

> Would be interested if maxconnections is a better use to throttle than upload limits or if this suggested parameter can cause issues.

Two solutions suggested in the above reddit link: 

_I've heard people reporting that they uploaded several TB in a few days when they have `peerblockfilters`_ -nullc

https://github.com/getumbrel/umbrel/blob/master/templates/bitcoin-sample.conf#L28

`maxuploadtarget` can also be used to manage upload limits

**Comment by fkorotkov** [comment]
@fanquake it's 12 hours since user creation. 

We've been battling with people creating fake accounts and using CIs available on GitHub including Cirrus for mining different crypto coins. 😪 This is just the first dump mechanism that prevents the bad actors to just re-create a new user right away.

Is it OK to wait for an hour so @apitko or you can re-run all the tasks on [this page](https://cirrus-ci.com/build/6161901701824512).

**Comment by wodry** [comment]
According to `bitcoind --help` version 0.21.0 `maxuploadtarget` is in MiB, so you should change MB to MiB and GB to GiB.

**Comment by apitko** [comment]
> According to `bitcoind --help` version 0.21.0 `maxuploadtarget` is in MiB, so you should change MB to MiB and GB to GiB.

Good catch! Will update to Mebi and Gibi.

**Comment by apitko** [comment]
Bee a while since I used Github! How can I edit a commit? i was only trying to submit a suggestion, was a bit tired Wanted to change MB to MiB and GB to GiB for clarity.

**Comment by fanquake** [comment]
@fkorotkov thanks for clarifying.

@apitko Take a look at https://github.com/bitcoin/bitcoin/blob/master/CONTRIBUTING.md#contributor-workflow.

**Comment by michaelfolkson** [comment]
@apitko: Make the additional change on your PR branch, commit that change, [squash](https://github.com/bitcoin/bitcoin/blob/master/CONTRIBUTING.md#squashing-commits) the two commits and then force git push (`git push -f`).

**Comment by jarolrod** [comment]
@apitko any update? Are you going to get back to this?

**Comment by apitko** [comment]
@jarolrod Had total hip replacement surgery and computer work went by the wayside. Jumping on this to button up tomorrow. Thanks!

**Comment by fanquake** [comment]
@apitko you'll need to [squash your commits](https://github.com/bitcoin/bitcoin/blob/master/CONTRIBUTING.md#squashing-commits). You can use `doc: add maxuploadtarget to bitcoinf.conf example` as the commit message.

**Comment by apitko** [comment]
@fanquake Thank you! Been awhile, a bit rusty with git.

File: share/examples/bitcoin.conf [hunk 1]
```diff
@@ -66,6 +66,10 @@
 # Maximum number of inbound+outbound connections.
 #maxconnections=
 
+#Maximum daily upload target in MiB per day (e.g. 'maxuploadtarget=1024' is 1GiB max throughput per day)
```
Comment (wodry) [inline share/examples/bitcoin.conf:69]: I'd like `1 GiB` better.

Comment (apitko) [reply share/examples/bitcoin.conf]: @wodry I like it better too. Let's see if I got my squashing and commit correct.

**Review by theStack — COMMENTED** [review]
Concept ACK - happy to review as soon as the commits are squashed into one :)

File: share/examples/bitcoin.conf [hunk 2]
```diff
@@ -66,6 +66,10 @@
 # Maximum number of inbound+outbound connections.
 #maxconnections=
 
+#Maximum daily upload target in MiB per day (e.g. 'maxuploadtarget=1024' is 1 GiB max throughput per day)
+#Limiting the free-for-all bandwidth uploads for those with bandwidth limits
+#maxuploadtarget=
```
Comment (jonatack) [inline share/examples/bitcoin.conf:71]: suggestion
```diff
-#Maximum daily upload target in MiB per day (e.g. 'maxuploadtarget=1024' is 1 GiB max throughput per day)
-#Limiting the free-for-all bandwidth uploads for those with bandwidth limits
+# Maximum upload bandwidth limit in MiB per day (e.g. 'maxuploadtarget=1024' is 1 GiB per day).
+# This limits the upload bandwidth for those with bandwidth limits. 0 = no limit (default: 0).
```

- "daily upload target  in MiB per day" is redundant (daily/per day)
- it's a limit, not a target
- mention the default and that 0 = no limit

Comment (jonatack) [reply share/examples/bitcoin.conf:71]: maybe mention that it does not apply to peers with "download" permission
```
$ bitcoind -help | grep -A3 maxuploadtarget=
  -maxuploadtarget=<n>
       Tries to keep outbound traffic under the given target (in MiB per 24h).
       Limit does not apply to peers with 'download' permission. 0 = no
       limit (default: 0)
```

Comment (apitko) [reply share/examples/bitcoin.conf:71]: Awesome! Will get to this shortly. I am having trouble squashing after I enabled 2FA and CLI won't allow me to authorize now to squash. I am also trying the Desktop version and does not give me a squash option, tried researching, any help appreciated.

**Review by luke-jr — COMMENTED** [review]
Maybe also reference doc/reduce-traffic.md in the comments?

**Comment by jarolrod** [comment]
pinging @apitko. If you can't or don't want to revisit this, you can close this PR and I will pick it up and squash.

**Comment by apitko** [comment]
@jarolrod Thank you Jarol! I've tried everything to squash the my commits using web, CLI and app based Git and it won't allow me with 2FA. Please feel free to pickup and squash. Frustration got the best of me.  And @jonatack had a good mention to add. Quoting @jonatack mention here:

$ bitcoind -help | grep -A3 maxuploadtarget=
  -maxuploadtarget=<n>
       Tries to keep outbound traffic under the given target (in MiB per 24h).
       Limit does not apply to peers with 'download' permission. 0 = no
       limit (default: 0)

**Comment by jarolrod** [comment]
@aptiko no problem! dont in #22080

**Comment by apitko** [comment]
@jarolrod You are awesome! Will close. Cheers!

**Pull request closed without merging**